docs/io/ascii/write.rst

```diff
@@ -287,15 +287,30 @@ has masked values.
   >>> table['Flux'] = MaskedColumn(table['Flux'], mask=[True, False])
   >>> table['magnitude'] = [u.Magnitude(25), u.Magnitude(-9)]
 
-Note that for columns with `~astropy.time.Time`, `~astropy.time.TimeDelta` and related values,
-the writer does not do any internal conversion or modification. These columns should be
-converted to regular columns with proper ``unit`` and ``name`` attribute before writing
-the table. Thus::
+For columns with `~astropy.time.Time` values and for columns that are thus, the values are
```

Comment:
I don't understand this sentence. Do you mean "columns that contain Time objects are are a time mix-in column?
How about
```suggestion
For columns holding `~astropy.time.Time` values, the values are
```
which can be read as "one (= the column is a Time object) or more (the column is a list of Time objects)".

Reply:
Ya, that's what I meant there. Your suggestion is far more spot on.